src/vivarium_census_prl_synth_pop/constants/results.py

```diff
@@ -95,5 +95,7 @@ def RESULTS_MAP(kind):
     df["key"] = columns
     df[
         "measure"
-    ] = kind  # per researcher feedback, this column is useful, even when it's identical for all rows
+    ] = (
+        kind
+    )  # per researcher feedback, this column is useful, even when it's identical for all rows
```

Comment:
Yuck. This formatting change (black?) hits code in the template and is ugly. Nothing for you to change here, you shouldn't need to change this block.